src/CMakeLists.txt

```diff
@@ -27,21 +27,29 @@ target_include_directories(hc PUBLIC
 	${HC_SRCDIR}
 	${Boost_INCLUDE_DIR}
 	${HC_EXTERNALDIR}/BamTools/src
+	${HC_EXTERNALDIR}/docopt.cpp
+	${PROJECT_BINARY_DIR}
 )
 target_link_libraries(hc
 	${Boost_LIBRARIES}
 )
 if (HC_STATIC)
-	target_link_libraries(hc BamTools-static)
+	target_link_libraries(hc BamTools-static docopt_s)
 else()
-	target_link_libraries(hc BamTools)
+	target_link_libraries(hc BamTools docopt_s)
```

Comment:
NOTE:
--------
I am mimicking the current behaviour where docopt is compiled as an object (*docopt.o*) straight into 
haploclique, no matter what shared/static setting is in action. No external shared object is ever needed.

Here the static docopt library is used no matter what, so that the final haploclique never relies on any lib docopt.so